notebooks/Tutorial_2_train_your_first_TTS_model.ipynb

```diff
@@ -0,0 +1,438 @@
+    " &emsp;| - udio2.wav<br /> \n",
+    " &emsp;| - audio3.wav<br /> \n",
+    "  ...<br /> \n",
+    "</span>\n",
+    "\n",
+    "and a **metdata.txt** file will have the audioname in parallel to the transcript, delimeted by `|`: \n",
+    " \n",
+    "<span style=\"color:purple;font-size:15px\">\n",
+    "# metadata.txt <br /> \n",
+    "audio1|This is my sentence. <br /> \n",
+    "audio2|This is maybe my sentence. <br /> \n",
+    "audio3|This is certainly my sentence. <br /> \n",
+    "audio4|Let this be your sentence. <br /> \n",
+    "...\n",
+    "</span>\n",
+    "\n",
+    "In the end, we should have the following **folder structure**:\n",
+    "\n",
+    "<span style=\"color:purple;font-size:15px\">\n",
+    "/MyTTSDataset <br /> \n",
+    "&emsp;| <br /> \n",
+    "&emsp;| -> metadata.txt<br /> \n",
+    "&emsp;| -> /wavs<br /> \n",
+    "&emsp;&emsp;| -> audio1.wav<br /> \n",
+    "&emsp;&emsp;| -> audio2.wav<br /> \n",
+    "&emsp;&emsp;| ...<br /> \n",
+    "</span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69501a10-3b53-4e75-ae66-90221d6f2271",
+   "metadata": {},
+   "source": [
+    "🐸TTS already provides tooling for the _LJSpeech_. if you use the same format, you can start training your models right away. <br /> \n",
+    "\n",
+    "After you collect and format your dataset, you need to check two things. Whether you need a **_formatter_** and a **_text_cleaner_**. <br /> The **_formatter_** loads the text file (created above) as a list and the **_text_cleaner_** performs a sequence of text normalization operations that converts the raw text into the spoken representation (e.g. converting numbers to text, acronyms, and symbols to the spoken format).\n",
+    "\n",
+    "If you use a different dataset format then the LJSpeech or the other public datasets that 🐸TTS supports, then you need to write your own **_formatter_** and  **_text_cleaner_**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e7f226c8-4e55-48fa-937b-8415d539b17c",
+   "metadata": {},
+   "source": [
+    "## ⏳️ Loading your dataset\n",
+    "Load one of the dataset supported by 🐸TTS.\n",
+    "\n",
+    "For this tutorial we will be using LJSpeech dataset.\n",
+    "We will start by defining dataset config and setting LJSpeech as our target dataset and define its path.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b3cb0191-b8fc-4158-bd26-8423c2a8ba66",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# BaseDatasetConfig: defines name, formatter and path of the dataset.\n",
+    "from TTS.tts.configs.shared_configs import BaseDatasetConfig\n",
+    "\n",
+    "output_path = \"tts_train_dir\"\n",
+    "if not os.path.exists(output_path):\n",
+    "    os.makedirs(output_path)\n",
+    "\n",
+    "dataset_config = BaseDatasetConfig(\n",
+    "    name=\"ljspeech\", meta_file_train=\"metadata.csv\", path=os.path.join(output_path, \"LJSpeech-1.1/\")\n",
```

Comment:
In the examples above the metadata file has a `.txt` extension.

Reply:
Oh that's a bug in the documentation as well. @erogol it should be `CSV` right?